mw-webapp/src/logic/reportsTable/columns.tsx

```diff
@@ -46,40 +48,63 @@ export const columns = [
     }),
   }),
   columnHelper.accessor<"jobsDone", JobDone[]>("jobsDone", {
-    header: "Jobs done",
+    header: "Jobs done (minutes)",
 
     /**
      * Cell with JobsDone items
      */
     cell: ({row}) => {
       return (
-        <div>
+        <div className={styles.cell}>
```

Comment:
maybe it will be ok to create separate component for table cell

Reply:
Created